_data-prepper/pipelines/configuration/processors/truncate.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: truncate
+parent: Processors
+grand_parent: Pipelines
+nav_order: 121
+---
+
+# truncate
+
+The `truncate` processor truncates a key's value at the beginning, the end or, on both sides of the value string based on processor's configuration. If the key's value is a list, then each member in the string list is truncated. Non-string members of the list are left untouched. When the `truncate_when` option is provided, the truncation of the input is done only when the condition specified is true for the event being processed.
+
+## Configuration
+
+You can configure the `truncate` processor using the following options.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`entries` | Yes | String list | A list of entries to add to an event.
+`source_keys` | Yes | String list | The list of sources keys that will be modified by the processor.
```

Comment:
This is not required anymore.

By default, it will truncate all fields.

> `source_keys` | Not | String list | The list of sources keys that will be modified by the processor. The default value is an empty list which indicates truncating all values.